src/formatDate.js

```diff
@@ -8,7 +8,51 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  let day = 0;
+  let month = 0;
+  let year = 0;
+
+  const splitDate = date.split(fromFormat[fromFormat.length - 1]);
```

Comment:
When splitting the date by the separator, it's assumed that the resulting array will have the date parts in the correct order. However, the code does not account for the possibility of the 'fromFormat' array being in a different order. Make sure to split and assign the date parts based on the order specified in the 'fromFormat' array.